cabal.project

```diff
@@ -44,7 +44,13 @@ import: cabal.project.blockio-uring
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/fs-sim
-  tag: 47879aa5edfd3a3f8824d61687e85b8f1586e010
+  tag: 235d5dd10aadb70defa5a2e843a4cadf5d6eb18e
```

Comment:
Updated to a newer hash of `fs-api` and `fs-sim` to permit building with `ghc-9.10.1` without an excessively large `allow-newer` stanza
.